Sources/UnleashProxyClientSwift/Client/UnleashProxyClientSwift.swift

```diff
@@ -122,6 +144,24 @@ public class UnleashClientBase {
         subscribe(name: event.rawValue, callback: callback)
     }
 
+    public func subscribe(_ event: UnleashEvent, callback: @escaping (Any?) -> Void) {
```

Comment:
Overloaded subscribe methods to allow us to pass event object to subscription handler. Debating with myself whether Any? is the correct type here, but it does mean that we can expand what we pass here in the future without adding additional overloads.